src/components/DappBrowser/Homepage.tsx

```diff
@@ -144,14 +145,30 @@ const Card = React.memo(function Card({
   goToUrl,
   site,
   showMenuButton,
+  index,
 }: {
   goToUrl: (url: string) => void;
   showMenuButton?: boolean;
   site: Omit<Site, 'timestamp'>;
+  index?: number;
 }) {
   const { isDarkMode } = useColorMode();
+  const browserHistoryStore = useBrowserHistoryStore();
   const { dapps } = useDapps();
 
+  const dappClickedBefore = useMemo(() => browserHistoryStore.hasVisited(site.url), [browserHistoryStore, site.url]);
+
+  const handlePress = useCallback(() => {
+    {
+      index &&
+        analyticsV2.track(analyticsV2.event.browserTrendingDappClicked, {
+          hasClickedBefore: dappClickedBefore,
+          index: index,
```

Comment:
why track index? i think it would be more relevant to track info about the dapp, such as name or url etc.

Reply:
happened to see this - I think the index is actually useful because it gives you an idea of the depth of engagement (whether the user scrolled beyond the first two cards)